internal/model/merchant/merchant_offer_table.go

```diff
@@ -0,0 +1,16 @@
+package merchant
```

Comment:
I'm personally not a fan of separating out interfaces into a seperate file from the functions themselves because it makes it difficult jump to function declarations during development